public/app/features/alerting/unified/components/rules/RuleListGroupView.tsx

```diff
@@ -28,6 +29,10 @@ export const RuleListGroupView: FC<Props> = ({ namespaces, expandAll }) => {
     ];
   }, [namespaces]);
 
+  useEffect(() => {
+    logInfo('loaded Alert Rules list');
```

Comment:
Maybe we could create an object with the log messages instead of using the string directly?. This could help us to gather all the messages we are tracking in the same place.

Instead of : 
`logInfo('loaded Alert Rules list');`
Have an object like: 
```
const logMessages = {
ruleListLoaded : 'loaded Alert Rules list',
....
```
and then 
`logInfo(logMessages.ruleListLoaded)
`